definitions/procedures/pulp/remove.rb

```diff
@@ -85,6 +87,55 @@ def remove_mongo
       end
     end
 
+    # rubocop:disable Metrics/BlockLength
+    def drop_migration_tables
+      with_spinner('Dropping migration tables') do
+        db_shell_input = <<-DBCOMMAND
+        BEGIN;
+        DROP TABLE IF EXISTS pulp_2to3_migration_migrationplan,
+          pulp_2to3_migration_pulp2blob,
+          pulp_2to3_migration_pulp2content,
+          pulp_2to3_migration_pulp2debcomponent,
+          pulp_2to3_migration_pulp2debcomponentpackage,
+          pulp_2to3_migration_pulp2debpackage,
+          pulp_2to3_migration_pulp2debrelease,
+          pulp_2to3_migration_pulp2debreleasearchitecture,
+          pulp_2to3_migration_pulp2distribution,
+          pulp_2to3_migration_pulp2distributor,
+          pulp_2to3_migration_pulp2distributor_pulp2_repos,
+          pulp_2to3_migration_pulp2erratum,
+          pulp_2to3_migration_pulp2importer,
+          pulp_2to3_migration_pulp2iso,
+          pulp_2to3_migration_pulp2lazycatalog,
+          pulp_2to3_migration_pulp2manifest,
+          pulp_2to3_migration_pulp2manifestlist,
+          pulp_2to3_migration_pulp2modulemd,
+          pulp_2to3_migration_pulp2modulemddefaults,
+          pulp_2to3_migration_pulp2packagecategory,
+          pulp_2to3_migration_pulp2packageenvironment,
+          pulp_2to3_migration_pulp2packagegroup,
+          pulp_2to3_migration_pulp2packagelangpacks,
+          pulp_2to3_migration_pulp2repocontent,
+          pulp_2to3_migration_pulp2repository,
+          pulp_2to3_migration_pulp2rpm,
+          pulp_2to3_migration_pulp2srpm,
+          pulp_2to3_migration_pulp2tag,
+          pulp_2to3_migration_pulp2yumrepometadatafile,
+          pulp_2to3_migration_reposetup CASCADE;
+        DROP SEQUENCE IF EXISTS pulp_2to3_migration_pulp2distributor_pulp2_repos_id_seq;
+        DELETE FROM django_migrations WHERE app = "pulp_2to3_migration";
+        DELETE FROM auth_permission WHERE content_type_id in (SELECT id FROM django_content_type WHERE app_label = 'pulp_2to3_migration');
+        DELETE FROM django_admin_log WHERE content_type_id in (SELECT id FROM django_content_type WHERE app_label = 'pulp_2to3_migration');
+        DELETE FROM django_content_type WHERE app_label = pulp_2to3_migration;
```

Comment:
Doesn't this line also need quotes?
```suggestion
        DELETE FROM django_content_type WHERE 'app_label = pulp_2to3_migration';
```

Reply:
Quoting the condition does not work, at least not for sending STDIN to the `pulpcore-manager dbshell` command. Perhaps we need to rethink using dbshell? I'm open to suggestions.